src/client.js

```diff
@@ -39,7 +38,7 @@ global.socket = initSocket();
 const component = (
   <Router render={(props) =>
         <ReduxAsyncConnect {...props} helpers={{client}} filter={item => !item.deferred} />
-      } history={history}>
+      } history={history} render={applyRouterMiddleware(useScroll())}>
```

Comment:
Yes. You've added the `render` property twice to the same element. Second overrides the first, I believe.